sceptre/plan/executor.py

```diff
@@ -7,41 +7,38 @@
 executing the command specified in a SceptrePlan.
 """
 import logging
-
 from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import List, Set
+
 from sceptre.plan.actions import StackActions
-from sceptre.stack_status import StackStatus
+from sceptre.stack import Stack
 
 
 class SceptrePlanExecutor(object):
 
-    def __init__(self, command, launch_order):
+    def __init__(self, command: str, launch_order: List[Set[Stack]]):
         """
-        Initalises a SceptrePlanExecutor, generates the launch order, threads
+        Initialises a SceptrePlanExecutor, generates the launch order, threads
         and intial Stack Statuses.
 
         :param command: The command to execute on the Stack.
-        :type command: str
 
-        :param launch_order: A list containing sets of Stacks that can be\
-                executed concurrently.
-        :type launch_order: list
+        :param launch_order: A list containing sets of Stacks that can be executed concurrently.
         """
 
         self.logger = logging.getLogger(__name__)
         self.command = command
         self.launch_order = launch_order
-
-        self.num_threads = len(max(launch_order, key=len))
-        self.stack_statuses = {stack: StackStatus.PENDING
-                               for batch in launch_order for stack in batch}
```

Comment:
This `stack_statuses` attribute was never used anywhere. I think it might be a relic of a former state of this code that has since been replaced and this was just left over.